### PR TITLE
fix: remove indent from generated query

### PR DESCRIPTION
**Pull request by RinkiyaKeDad**
I think I have found a fix for issue #1965. The indentation issue would have also come when using subscriptions so I've tried to fix that too. 

I could not test my changes. I followed the written instructions but had the following problem:
1. First I changed the path of extensions.graphback.model in the .graphqlrc file,
ss1
2. Then I ran the command in the templates folder and received this error,
ss2

While I wasn't able to test it I think the changes I've made should fix the issue.

### CODE REVIEW

**Comment by machi1990** [comment]
@RinkiyaKeDad amazing change and nice PR description, thanks. @craicoverflow mind having a look at this one?

**Comment by machi1990** [comment]
@RinkiyaKeDad the build: https://github.com/aerogear/graphback/runs/1165819855?check_suite_focus=true if failing and it is related to the change you made. 

Do you mind running `yarn test -u` in the [packages/graphback-codegen-client/](https://github.com/aerogear/graphback/blob/b16aa6011c4ee00d00be3120ce27d28a01fad99d/packages/graphback-codegen-client/) directory to update the snapshot? Thanks.

**Comment by wtrocki** [comment]
Looks good. Please update snapshots

**Comment by RinkiyaKeDad** [comment]
Hey @machi1990 I ran the command you told in order to update the snapshots. Recevied this error:
ss3
Please help me. Thanks.

**Comment by wtrocki** [comment]
> Please help me. Thanks

Every repo has contributimg.md file that everyone should follow. it is not perfect but it has all info and it will resolve problems. It is ok to ask maintainers to solve problems with code, but we cannot solve problems with your environment etc.

**Comment by RinkiyaKeDad** [comment]
Really sorry @wtrocki. Didn't know this :sweat_smile:. Will take care from next time.

**Comment by wtrocki** [comment]
File in question: https://github.com/aerogear/graphback/blob/master/CONTRIBUTING.md
If you follow steps your problem will be resolved.

**Comment by RinkiyaKeDad** [comment]
Can you please just tell me if this is a  problem with my environment? 
ss4

I see that the package.json of graphback-codegen-client mentions ```@graphback/core``` as a dependency and just be sure I even ran ```yarn install``` in the graphback-codegen-client directory where it gave me a success message saying "Already up-to-date". This means that ```@graphback/core``` should be present in node_modules but then why is it having trouble importing it?

**Comment by machi1990** [comment]
> Can you please just tell me if this is a problem with my environment?
> ss4
> 
> I see that the package.json of graphback-codegen-client mentions `@graphback/core` as a dependency and just be sure I even ran `yarn install` in the graphback-codegen-client directory where it gave me a success message saying "Already up-to-date". This means that `@graphback/core` should be present in node_modules but then why is it having trouble importing it?

Thanks for following up on this one. The error you are seeing is because the project was not built. Can you run the below command in the project root? 

```sh
yarn
yarn build
```

Once that is done, you can run the `yarn test -u` command in the `graphback-codegen-client` package. 

Let me know if that work.

**Comment by wtrocki** [comment]
https://github.com/aerogear/graphback/blob/master/CONTRIBUTING.md#building

**Comment by RinkiyaKeDad** [comment]
Sorry, I only ran ```yarn install``` after reading the CONTRIBUTING.md since I was under the assumption that building is something you only did before deployment :sweat_smile: Thanks a lot for helping me out :)

File: packages/graphback-codegen-client/src/templates/gqlTemplates.ts [hunk 1]
```diff
@@ -28,7 +28,8 @@ export const findOneQuery = (t: ModelDefinition) => {
     ${fieldName}(id: $id) {
       ...${t.graphqlType.name}ExpandedFields
     }
-  }`
+}
+`
```
Comment (craicoverflow) [inline packages/graphback-codegen-client/src/templates/gqlTemplates.ts:32]: This fixes the original problem, so thank you. This also introduces an extra blank line too though:

image

**Review by craicoverflow — CHANGES_REQUESTED** [review]
This fixes the original problem, so thank you. This also introduces an extra blank line too though:

image

File: packages/graphback-codegen-client/src/templates/gqlTemplates.ts (hunk 1 above)
Comment (RinkiyaKeDad) [reply packages/graphback-codegen-client/src/templates/gqlTemplates.ts:32]: @craicoverflow thanks for reviewing. This is happening for the generated mutations too, so should I fix this everywhere?
ssToday

Comment (craicoverflow) [reply packages/graphback-codegen-client/src/templates/gqlTemplates.ts:32]: Sure! If you see other area that are wrong you can fix those too and we can review the code.

**Review by craicoverflow — APPROVED** [review]
Thank

**Pull request merged**